### DISCUSSION TITLE
How can I feed a top-level CUE value into an expression during export?

### DISCUSSION THREAD

**Discussion by seh** [cuelang/cue#1054] [Q&A]
Suppose I have CUE file with the following content:
```cue
[
	{
		a: 1
	},
	{
		b: 2
	},
]
```
It's a top-level list, not a struct with any field names.

Suppose I'd like to export it as [a multi-document YAML stream](https://pkg.go.dev/cuelang.org/go/pkg/encoding/yaml#MarshalStream). Ideally I'd be able to refer to this top-level value in a command like this:
> _cue export list.cue --out=text --expression='yaml.MarshalStream($)'_

Here I used the identifier `$` in an appeal to Jsonnet, as some way to refer to the top-level value, but of course `$` is not understood by CUE.

I experimented with using the `--path` and `--with-context` flags, to no avail.

**Comment by cueckoo** [comment]
This discussion has been migrated to https://github.com/cue-lang/cue/issues/1054.

For more details about CUE's migration to a new home, please see https://github.com/cue-lang/cue/issues/1078.
